src/internal/transforms/sqlingest.go

```diff
@@ -96,10 +100,48 @@ func makeWriterFactory(formatName string) (writerFactory, error) {
 		}
 	case "csv":
 		factory = func(w io.Writer, fieldNames []string) sdata.TupleWriter {
-			return sdata.NewCSVWriter(w, fieldNames)
+			return sdata.NewCSVWriter(w, nil)
```

Comment:
is this change intentional?

Reply:
Yeah, setting the csv writer fields to nil removes the header.  I don't really see the benefit of the header since the user will know what the asked for and it makes splitting and concatenating files more difficult.

Reply:
makes sense 👍